Cargo.toml

```diff
@@ -27,10 +28,10 @@ directories = "5.0"
 dotenvy = "0.15"
 futures = "0.3"
 indexmap = { version = "2.0", features = ["serde"] }
-k8s-openapi = { version = "0.20", default-features = false, features = [
+k8s-openapi = { version = "0.21", default-features = false, features = [
```

Comment:
FYI https://github.com/stackabletech/operator-rs/pull/769. Not saying we should block this though

Reply:
Yeah we can wait for the `stackable-operator` release to bump `k8s-openapi` and friends.